ui/src/app/cohort-review/detail-tab-table/detail-tab-table.component.tsx

```diff
@@ -469,12 +486,15 @@ export const DetailTabTable = withCurrentWorkspace()(
     textFilter(column: string) {
       const {domain, filterState} = this.props;
       const columnFilters = filterState.tabs[domain];
+      const filtered = !!columnFilters[column];
       let fl: any, ip: any;
       return <span>
-        <i className='pi pi-filter' onClick={(e) => {
-          fl.toggle(e);
-          ip.focus();
-        }}/>
+        <i className='pi pi-filter'
+          style={filtered ? filterIcons.active : filterIcons.default}
+          onClick={(e) => {
+            fl.toggle(e);
+            ip.focus();
+          }}/>
```

Comment:
Is there any way to centralize this logic? Seems that we are using similar logic above(line 461) and in table-page.tsx(line 437) as well.

Reply:
Possibly, but i think it's simpler to leave it since all three are slightly different.

 (This one calls both `fl.toggle(e)` and  `ip.focus()` every time, above calls only `fl.toggle(e)`, and `table-page.tsx` calls `fl.toggle(e)` every time and `ip.focus()` sometimes)